docs/guide/content_rendering/templates/template_configuration.md

```diff
@@ -0,0 +1,77 @@
+# Template configuration
+
+Template usage is configured under the `content_view` key.
+
+The following example configuration defines template usage for several cases:
+
+``` yaml
+[[= include_file('code_samples/front/render_content/config/packages/views.yaml', 4, 8) =]][[= include_file('code_samples/front/render_content/config/packages/views.yaml', 9, 29) =]]
+```
+
+## Scope
+
+The content view configuration must be placed under `ezplatform.system.<scope>`.
+
+Scope defines the SiteAccesses for which the configuration is valid.
+It may be a SiteAccess, a SiteAccess group, or one of the [generic configuration scopes](../../siteaccess.md#scope).
+
+## Page layout
+
+``` yaml
+[[= include_file('code_samples/front/render_content/config/packages/views.yaml', 7, 8) =]]
+```
+
+`pagelayout` defines the general layout of the whole site.
+Other templates can [extend the pagelayout](#page-layout).
+
```

Comment:
I suggest change the order: first description, then codeblock like in the View types.